web_src/less/helpers.less

```diff
@@ -175,4 +175,5 @@
 @media @mediaSm {
   .db-small { display: block !important; }
   .w-100-small { width: 100% !important; }
+  .js-small { justify-content: flex-start !important; }
```

Comment:
According to the scheme above: Shouldn't this be `jf-small`?

Reply:
I took the scheme from the existing `justify-content: flex-start` https://github.com/go-gitea/gitea/blob/94d6d93cc29dd8837d87e1aa58de8f52676d4963/web_src/less/helpers.less#L11

Reply:
It's slightly confusing naming because we usually use `.js` prefix for JavaScript stuff, but I don't have a better idea as it does follow the scheme.